Tabloid/client/src/components/Tag.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Card, CardBody } from "reactstrap";
+
+const Tag = ({ tag }) => {
+  return (
+    <Card >
+      <p className="text-left px-2">{tag.name}</p>
+      <CardBody>
+        <iframe className="tag"
+          src={tag.url}
+          title="YouTube tag player"
+          frameBorder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowFullScreen />
+
+        <p>
```

Comment:
The iframe code is left-over from what the video component used to display inline YouTube videos, it doesn't make any sense here in tags, please remove